src/SDKs/CognitiveServices/dataPlane/Search/BingCustomImageSearch/BingCustomImageSearch/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides API functions for consuming the Microsoft Azure Cognitive Services Custom Search API.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Please bump the assemblyversion and update packagereleasenotes in csproj

Reply:
The last published Nuget version was `1.0.0`, I think `AssemblyFileVersion` can be left as it is, please update the version in corresponding csproj